pyproject.toml

```diff
@@ -88,6 +88,7 @@ devel-test = [
   "pytest-bdd        ~=7.0",
   "pytest-cov        ~=5.0",
   "testcontainers    ~=3.0",
+  "jsonpath-ng       ~=1.6",
```

Comment:
I might be naive, but why use `jsonpath-ng`? Are there really complicated paths we need to handle?

While I'm not entirely opposed to having new dependencies as need be (especially in tests), I'm curious whether a simple `path.split(".")` might be sufficient?

Reply:
I wasn't sure how sophisticated the paths could get. I went with the option that allowed for complicated paths, but since this is really only used in implementing the features, it's probably overkill. I'll remove